### ISSUE TITLE
Is TSS enrichment plot showing the correct output?

### ISSUE THREAD

**Issue by kkupkova** [deeptools/deepTools#1090]
**Welcome to deepTools GitHub repository! Before opening the issue please check
that the following requirements are met :**

 - [ ] Search whether this issue (or a similar issue) has been solved before
 using the search tab above. Link the previous issue if appropriate below.

 - [ ] Paste your deepTools version (`deeptools --version`) and your python
 version (`python --version`) below.

 - [ ] Paste the full deepTools command that produces the issue below
 (ignore if you simply spotted the issue in the code/documentation).

 - [ ] Paste the output printed on screen from the command that produces the issue
 below (ignore if you simply spotted the issue in the code/documentation).

-----
Hi! I have deeptools 3.3.1 / Python 3.7.3 and I am not reporting an error, rather concern. I have been struggling to replicate the results from --referencePoint TSS when I generate my custom list of TSSs (just 1 bp) and run computeMatrix reference-point with a, b set to 2000. I was able to replicate the results when I generated a new list of TSSs, where I didn't take into account the strand. Shouldn't a TSS of a gene on "-" strand be in fact its end coordinate?

**Comment by dpryan79** [comment]
Yes, the strand is taken into account, so please double check that when you manually generated 1-base BED file that you did in fact get the correct coordinates. You can also check the signals reported by `gunzip`ping the output and having a look (it's just a text file).
